scripts/ci/ci_pylint.sh

```diff
@@ -17,13 +17,14 @@
 # limitations under the License.
 #
 
-# Script to run Pylint on all code. Should be started from root directory:
-# ./[airflow dir]/scripts/ci/ci_pylint.sh
+# Script to run Pylint on all code. Should be started from root directory of Airflow:
+# ./scripts/ci/ci_pylint.sh
 
 set -ex
 
 find . -name "*.py" \
 -not -path "./.eggs/*" \
 -not -path "./airflow/www/node_modules/*" \
 -not -path "./airflow/_vendor/*" \
--not -path "./build" | grep -vFf scripts/ci/pylint_todo.txt | xargs pylint --output-format=colorized
+-not -path "./build/*" \
+! -name 'webserver_config.py' | grep -vFf scripts/ci/pylint_todo.txt | xargs pylint --output-format=colorized
```

Comment:
To be consistent with the other exclusions:

```suggestion
-not -name 'webserver_config.py' | grep -vFf scripts/ci/pylint_todo.txt | xargs pylint --output-format=colorized
```